src/qtgui/dockfft.cpp

```diff
@@ -352,9 +359,13 @@ void DockFft::readSettings(QSettings *settings)
         setFftSize(intval);
     emit fftSizeChanged(fftSize());
 
-    intval = settings->value("fft_window", DEFAULT_FFT_WINDOW).toInt(&conv_ok);
-    if (conv_ok)
-        ui->fftWinComboBox->setCurrentIndex(intval);
+    strval = settings->value("fft_window", "hann").toString();
+    auto it = std::find(window_strs.begin(), window_strs.end(), strval);
+    if (it == window_strs.end())
+        intval = DEFAULT_FFT_WINDOW;
```

Comment:
Do you suppose we should fall back to integer decoding for backwards compatibility with earlier versions of Gqrx?

Reply:
Even better, increment `configversion` to 4 and read the setting like so:

https://github.com/gqrx-sdr/gqrx/blob/282d8559332db7b0d424ae6acbb5edc1af9b22fa/src/qtgui/dockrxopt.cpp#L474-L480

Reply:
I was thinking about options for this, and whether it was worth the code to save the user a click. I'm on the road for a few and don't want to rush a solution ... but feel free to add something if you want to get a release out.

Reply:
I added a fallback in 91a42eea657f53567f90499f35bfbe2d4b73f9f1.